charmhelpers/fetch/ubuntu.py

```diff
@@ -738,6 +799,74 @@ def _verify_is_ubuntu_rel(release, os_release):
             'version ({})'.format(release, os_release, ubuntu_rel))
 
 
+def _add_bare_openstack(openstack_release):
+    """Add cloud or distro based on the release given.
+
+    The spec given is, say, 'ussuri', but this could apply cloud:bionic-ussuri
+    or 'distro' depending on whether the ubuntu release is bionic or focal.
+
+    :param openstack_release: the OpenStack codename to determine the release
+        for.
+    :type openstack_release: str
+    :raises: SourceConfigError
+    """
+    # So it's distro, which means it's a no-op so end here.
```

Comment:
Should this comment be moved somewhere else? It's not necessarily "distro" at this point. I suspect this line was moved by mistake from somewhere else?

Reply:
Ack; left in by mistake.